.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
```

Comment:
I feel like this table can be omitted. The whole bug report is descriptive as it is, plus github has a handy sidebar for handling labels, assignees, etc.